options/rtdl/generic/linker.cpp

```diff
@@ -130,20 +134,72 @@ SharedObject *ObjectRepository::requestObjectWithName(frg::string_view name,
 		return fd;
 	};
 
+	// TODO(arsen): this process can probably undergo heavy optimization, by
+	// preprocessing the rpath only once on parse
+	auto processRpath = [&] (frg::string_view &path) {
```

Comment:
Do not take a view by reference. (This is not changed inside the lambda, either.)